tools/mo/openvino/tools/mo/back/ConvolutionNormalizer.py

```diff
@@ -29,22 +33,69 @@ def resolve_convolution_with_group(node: Node, group: int, ir_version: str):
+            weights_node = node.in_port(1).get_source().node
+            input_node = node.in_port(0).get_source().node
+
+            weights_shape = Shape(node.graph, {'name': node_name + '/ShapeOfWeights'}).create_node()
+            weights_shape.in_port(0).connect(weights_node.out_port(0))
+
+            weights_spatial_shape = create_op_with_const_inputs(node.graph, StridedSlice,
+                                                                port_value_dict={1: int64_array([2]),
+                                                                                 2: int64_array([-1])},
+                                                                op_attrs={'begin_mask': [1],
+                                                                          'end_mask': [0],
+                                                                          'new_axis_mask': [0],
+                                                                          'shrink_axis_mask': [0],
+                                                                          'ellipsis_mask': [0]},
+                                                                input_node=weights_shape)
+
+            const_part_of_shape = Const(node.graph, attrs=dict(name=node_name + '/GroupsAndOutputChannelsSize',
+                                                               value=shape_array(
+                                                                   [group, node.output // group]))).create_node()
+
+            input_shape_node = Shape(node.graph, {'name': node_name + '/ShapeOfInput'}).create_node()
+            input_shape_node.in_port(0).connect(input_node.out_port(0))
+
+            input_num_channels = create_op_with_const_inputs(node.graph, Gather,
+                                                             port_value_dict={1: int64_array([1]), 2: int64_array(0)},
+                                                             op_attrs={'name': node_name + '/GatherInputNumChannels'},
+                                                             input_node=input_shape_node)
+
+            # input channels num divided by number of groups to alight weights shape into [GROUPS C_OUT C_IN X Y]
+            C_IN = create_op_with_const_inputs(node.graph, Div,
+                                               port_value_dict={1: int64_array(group)},
+                                               op_attrs={'name': node_name + '/Div'},
+                                               input_node=input_num_channels)
+
+            new_shape_node = Concat(node.graph, {'axis': 0, 'in_ports_count': 3}).create_node()
+            new_shape_node.in_port(0).connect(const_part_of_shape.out_port(0))
+            new_shape_node.in_port(1).connect(C_IN.out_port(0))
+            new_shape_node.in_port(2).connect(weights_spatial_shape.out_port(0))
+            reshape = Reshape(node.graph, {'override_output_shape': True}).create_node()
```

Comment:
please set special_zero that is mandatory attribute by the spec

Reply:
done